src/components/helpers/import.ts

```diff
@@ -0,0 +1,18 @@
+import { Account } from '@solana/web3.js';
+import * as bs58 from 'bs58';
+
+/**
+ * Returns an account object when given the private key
+ *
+ */
+export const decodeAccount = (privateKey: string) => {
+    try {
+        return new Account(JSON.parse(privateKey));
+    } catch (_) {
+        try {
+           return new Account(bs58.decode(privateKey));
+        } catch (_) {
+           return undefined;
+        }
+    }
+}
```

Comment:
Also I know this isn't really part of your feature, but it would be great if there's an error message to the user, like "Invalid file format." if both decodes fail.

Reply:
I think that this may not be necessary because currently the button to complete the import is not clickable until the import works.  [see code](https://github.com/project-serum/spl-token-wallet/blob/master/src/components/AddAccountDialog.js#L77)